django/db/backends/base/operations.py

```diff
@@ -517,12 +518,20 @@ def convert_durationfield_value(self, value, expression, context):
         return value
 
     def check_aggregate_support(self, aggregate_func):
-        """Check that the backend supports the provided aggregate
+        warnings.warn(
+            "check_aggregate_support has been deprecated. Use "
+            "check_expression_support instead.",
+            RemovedInDjango20Warning, stacklevel=2)
+        return self.check_expression_support(aggregate_func)
 
-        This is used on specific backends to rule out known aggregates
-        that are known to have faulty implementations. If the named
-        aggregate function has a known problem, the backend should
-        raise NotImplementedError.
+    def check_expression_support(self, expression):
+        """
+        Check that the backend supports the provided expression.
```

Comment:
Shall I change this to "Checks.." to keep within the current style guide? If 24175 is accepted, then it can update occurrences to "Check..". Otherwise, I can leave it as-is. The existing docstring was "Check that the backend.." if that helps at all.